src/graphql-schema/resolvers.js

```diff
@@ -25,9 +26,19 @@ module.exports = function Resolvers (sbot) {
     }
   }
 
+  function getProfileDb2 (opts, cb) {
+    const { feedId } = opts
+
+    sbot.aboutSelf.get(feedId, (err, profile) => {
+      if (profile.publicWebhosting !== true) cb(null, null)
```

Comment:
I think you might want `cb(null, {})` for this case. will make graphql stuff work better. We'll see